src/Fixer/Phpdoc/GeneralPhpdocAnnotationRemoveFixer.php

```diff
@@ -105,7 +104,6 @@ protected function createConfigurationDefinition()
         $annotations = new FixerOptionBuilder('annotations', 'List of annotations to remove, e.g. `["author"]`.');
         $annotations = $annotations
             ->setAllowedTypes(array('array'))
-            ->setDefault(array())
```

Comment:
looks like it could work without:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/src/Fixer/Basic/Psr0Fixer.php#L95
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/tests/Fixer/Basic/Psr0FixerTest.php#L59


actually, only `header_comment` throws `RequiredFixerConfigurationException` intentionally, so only this fixer require config. in `psr0` there is indeed missing of default value for `dir` config key, but if one won't pass any configuration (`psr0 => true`), then configs are not resolved and everything works properly. sure, we shall add that default empty value for clarity